includes/gateways/paypal/includes/settings-paypal.php

```diff
@@ -50,6 +50,13 @@
 		'default'     => 'no',
 		'description' => sprintf( __( 'Log PayPal events, such as IPN requests, inside %s', 'woocommerce' ), '<code>' . WC_Log_Handler_File::get_log_file_path( 'paypal' ) . '</code>' ),
 	),
+	'ipn_notification' => array(
+		'title'       => __( 'IPN Email Notifications', 'woocommerce' ),
+		'type'        => 'checkbox',
+		'label'       => __( 'Enable IPN email notification', 'woocommerce' ),
+		'default'     => 'yes',
+		'description' => __( 'Sends an admin email each time an IPN is received from PayPal and is related to cancelled order, refunded order, payment reversal, cancelled reversal.', 'woocommerce' )
```

Comment:
I think this wording is kind-of confusing (and not 100% accurate since the email gets sent to whoever is handling the orders). How about just "Send a notification to the user handling orders when an IPN is received from PayPal."?